v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single [transaction](https://www.cockroachlabs.com/docs/dev/transactions.html). Executing statements in parallel helps reduce aggregate latency and improve performance. 
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+SQL engines traditionally execute the SQL statements sequentially. The server executes each statement to completion and sends the return value of each statement to the client. Only after the client receives the return value of a statement, it sends the next SQL statement to be executed. 
+
+In the case of a traditional single-node SQL database, statements are executed on the single machine, and so the execution does not result in any communication latency. However, in the case of a globally-distributed and replicated database like CockroachDB, execution of statements can span multiple nodes. The coordination between nodes results in communication latency. Executing SQL statements sequentially results in higher cumulative latency.
+
+With parallel statement execution, however, multiple SQL statements within a transaction are executed at the same time, thereby reducing the aggregate latency over multiple SQL statements. 
+
+## When to use Parallel Statement Execution
```

Comment:
nit: capitalize "Use"

Reply:
Also, I think this should be combined with the section on Independent Statements. See my comment in that section.

Reply:
Fixed.